compile.go

```diff
@@ -88,10 +88,6 @@ var (
 	prelude = []string{
 		"universe",
 		"influxdata/influxdb",
-		"math",
```

Comment:
@aanthony1243 removed those because otherwise "math.abs" would not perform as expected in `universe.flux`. Has to do with "math" should be imported first but sometimes it would be imported after because maps are processed randomly.

Reply:
there was a data race wrt the order the packages were loaded.   moving them to `universe.flux` has the same outcome